src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/LocalPackageModel.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using NuGet.Packaging.Core;
+using NuGet.Protocol;
+using NuGet.VisualStudio.Internal.Contracts;
+
+namespace NuGet.PackageManagement.UI
+{
+    class LocalPackageModel : PackageModel
+    {
+        public LocalPackageModel(PackageIdentity identity,
+            string packagePath,
+            IVulnerable vulnerableCapability,
+            string? title = null,
+            string? description = null,
+            string? authors = null,
+            Uri? projectUrl = null,
+            string[]? tags = null,
+            string? copyright = null)
+            : base(identity, title, description, authors, projectUrl, tags, copyright)
+        {
+            PackagePath = packagePath;
+            VulnerableCapability = vulnerableCapability;
+        }
+
+        public string PackagePath { get; }
+        IVulnerable VulnerableCapability { get; }
+
+        public IReadOnlyList<PackageVulnerabilityMetadataContextInfo> GetPackageVulnerabilities()
+        {
+            return VulnerableCapability.Vulnerabilities;
+        }
+
+        public bool IsPackageVulnerable()
+        {
+            return VulnerableCapability.IsVulnerable;
+        }
+
+        public PackageVulnerabilitySeverity GetPackageVulnerabilityMaxSeverity()
+        {
+            return VulnerableCapability.VulnerabilityMaxSeverity;
+        }
```

Comment:
More concise, especially useful as we start having more capabilities.

```suggestion
        public IReadOnlyList<PackageVulnerabilityMetadataContextInfo> GetPackageVulnerabilities() => VulnerableCapability.Vulnerabilities;

        public bool IsPackageVulnerable() => VulnerableCapability.IsVulnerable;

        public PackageVulnerabilitySeverity GetPackageVulnerabilityMaxSeverity() => VulnerableCapability.VulnerabilityMaxSeverity;
```